### ISSUE TITLE
some skins voiceover doesnt work

### ISSUE THREAD

**Issue by enescglyn** [B3akers/LeagueSkinChanger#40]
There are a lot of skins that have their own voiceovers, when using lolskin all of them work but with leagueskin changer only some work

for example:
senna orange skin - has voiceovers but uses original ones
elementalist lux - has voiceovers does work
dark cosmic lux - has voiceovers but uses original ones (if i remember correctly)

**Comment by B3akers** [comment]
Yeah i noticed that some skins uses original voice overs or doesnt work at all, when fix i will close that issue

**Comment by B3akers** [comment]
I checked issue, problem is that on server you still have default skin, some skins have special VO and it's different packet, for example LeeSin has "Laugh3D" (like most skins/champions) but Storm Dragon Lee Sin has "Laugh3D" (which is empty sound) and "Laugh3DGeneral" which is send by server in diffrent packet since for server you have still default skin it will not send packet to play "Laugh3DGeneral", that is why in some skins VO doesn't work, or just play original, i can fix it by emulating server but then i have to add VO manually for each skin which doesn't work

**Comment by enescglyn** [comment]
Hello, do you require to own the skin to get the voiceover packets like
Laugh3Dgeneral? if no then would it be possible for you to make a system
where we get all voiceovers and tailor them at our own to make voiceovers
work?

B3akers <notifications@github.com>, 29 Oca 2021 Cum, 04:11 tarihinde şunu
yazdı:

**Comment by B3akers** [comment]
I'm looking for a way to dump all custom VO, because doing it manually doesn't make sense

**Comment by zola1919** [comment]
Can't wait for it, would really love the ult on nightbringer yas!

**Comment by enescglyn** [comment]
I recommend the maker of this skin changer to look at lolskin's source to find how they dump voiceover names. I have heard people begging for skins/chromas to be added so it might be manual too.
